5.5/contrib/common.sh

```diff
@@ -134,23 +146,19 @@ function wait_for_mysql_master() {
 # mysql_master_addr lookups the 'mysql-master' DNS and get list of the available
 # endpoints. Each endpoint is a MySQL container with the 'master' MySQL running.
 function mysql_master_addr() {
-  local service_name=${MYSQL_MASTER_SERVICE_NAME:-mysql-master}
-  local endpoints=$(dig ${service_name} A +search +short 2>/dev/null)
-  # FIXME: This is for debugging (docker run)
-  if [ -v MYSQL_MASTER_IP ]; then
-    endpoints=${MYSQL_MASTER_IP-}
-  fi
-  echo -n "$(echo $endpoints | cut -d ' ' -f 1)"
+  echo -n "${MYSQL_MASTER_SERVICE_NAME:-mysql-master}"
```

Comment:
I guess it was partly laziness and partly to keep it in common.sh. But I now I think I'll get rid of the function. Will rework.